plugins/aws_sts/lib/samson_aws_sts/samson_plugin.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module SamsonAwsSts
+  SESSION_DURATION_MIN = 900  # 15 minutes
+  SESSION_DURATION_MAX = 3600 # 1 hour
```

Comment:
would bump that since deploy could be long ... maybe 2 hours or max deploy length `Rails.application.config.samson.deploy_timeout`